src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more about how to work with
+projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board] hosts issues, bugs and features which are not a part of a particular release board. While each issue is opened in
+its corresponding repository, this board gives an aggregated view of all open issues across all repositories. The board has total of seven
+columns. Any new issues or epics should typically be assigned to the `Backlog` column. Epics which are candidates to be included in
+the next release are moved to the `Next Version Candidate` column. During [planning](#planning), all epics from the `Next
+Version Candidate` column are reviewed and triaged. Epics which are selected to be in the next release are moved to
+the [current release board](#current-release-board). The next 3 columns sort issues by priority. All unassigned issues are sorted into these
+three columns based on the priority discussed in the team meeting. Issues from these columns are ready to be worked on. Any leftover work
+from the previous release goes to the `Work in Progress column`. The `Close?` column holds any issues/epics that can probably be closed for
```

Comment:
> The `Close?` column...

Are you waiting for any inputs from someone?

Reply:
I just wanted to give some time to folks to review. I think we can merge now

Reply:
Okay. So do you want to remove the "?"

Reply:
This is the name of the column on the Backlog board - `Close?`, so it included in the description along side with the other columns description